modules/kobalt-plugin-api/src/main/kotlin/com/beust/kobalt/internal/TaskManager.kt

```diff
@@ -79,7 +79,17 @@ class TaskManager @Inject constructor(val args: Args,
     }
 
     fun runTargets(passedTaskNames: List<String>, allProjects: List<Project>): RunTargetResult {
-        val taskInfos = calculateDependentTaskNames(passedTaskNames, allProjects)
+        // Check whether tasks passed at command line exist
+        passedTaskNames.forEach {
+            if (!hasTask(TaskInfo(it)))
+                throw KobaltException("Unknown task: $it")
```

Comment:
This check is already performed in `TaskManager.createGraph2()` :

```
        //
        // Keep only the tasks we need to run.
        //
        val taskInfos = passedTasks.filter {
            if (!nodeMap.keys().contains(it.taskName)) {
                throw KobaltException("Unknown task: $it")
            }
            it.matches(projectName)
        }
```

Why do it in two different places?